### PR TITLE
Make compatible with Python 3.5.

### PR DESCRIPTION
**Pull request by symbolist**

### CODE REVIEW

File: blockstore/apps/bundles/store.py [hunk 1]
```diff
@@ -647,7 +648,7 @@ def is_safe_file_path(path):
 
 def create_hash(start_data=b''):
     """Whenever there's a need for hashing, we use 20-byte BLAKE2b."""
-    return hashlib.blake2b(start_data, digest_size=20)  # pylint: disable=no-member
```
Comment (symbolist) [inline blockstore/apps/bundles/store.py]: Is not available in Python 3.5.

**Comment by symbolist** [comment]
@ormsbee I have made all the necessary changes if you would like to take a look (otherwise will have someone from LX review tomorrow). Still investigating why quality is failing on CircleCI.

**Comment by ormsbee** [comment]
The changes look fine to me, but it looks like quality is failing to run altogether now?

**Comment by symbolist** [comment]
@ormsbee Had to pin `astroid<1.6` to get it to run. Ran the job twice and it worked both times.

**Comment by symbolist** [comment]
FYI @bradenmacdonald since you are using some of the type stuff in 3.6. :)

**Comment by bradenmacdonald** [comment]
Hmm, that kinda sucks that we can't use f-strings and some of these other things. But thanks for making this change @symbolist.

**Comment by edx-webhook** [comment]
Although this pull request is already merged, I've created [OSPR-5449](https://openedx.atlassian.net/browse/OSPR-5449) so that we can track it in Jira.

There is nothing you have to do. No action is needed from your side.  Thanks again for your contribution.

**Pull request merged**